MPF.Core/Modules/BaseParameters.cs

```diff
@@ -1209,6 +1209,60 @@ private static string RemoveHexIdentifier(string value)
             }
         }
 
+        /// <summary>
+        /// Get a isobuster-formatted PVD from a PS3 ISO, if possible
+        /// </summary>
+        /// <param name="isoPath">Path to ISO file</param>
+        /// <param name="pvd">Formatted PVD string, otherwise null</param>
+        /// <returns>True if PVD was successfully parsed, otherwise false</returns>
+        protected static bool GetPVD(string isoPath, out string? pvd)
+        {
+            pvd = null;
+            try
+            {
+                // Get PVD bytes from ISO file
+                var buf = new byte[96];
+                using (FileStream iso = File.OpenRead(isoPath))
+                {
+                    // TODO: Don't hardcode 0x8320
```

Comment:
I know this is a copy-paste of the original code, but how else would this data be found other than hardcoding the sector offset?

Reply:
There's a way to do it properly, reading the volume descriptors (until Volume Descriptor Set Terminator is reached, Type=255), and when Type=1 is found you're reading the PVD. More here: https://wiki.osdev.org/ISO_9660#Volume_Descriptors
But it's 0x8320 for all the images I've dealt with so as long as you know a PVD exists when calling this function..